docs/pages/build/submitting-to-app-stores.md

```diff
@@ -0,0 +1,105 @@
+
+   - You will be asked to sign into your Apple Developer account and select your team. You can also provide these information by using `--apple-id`, `--apple-team-id` params and `EXPO_APPLE_PASSWORD` environment variable.
+   - The command will look for `expo.ios.bundleIdentifier` in your app configuration, or you can provide it directly using `--bundle-identifier` command param
+   - If you are submitting your app for the first time, it will be automatically created.
+     Unless `expo.name` in your app configuration is found or `--app-name` param is provided, you will be prompted for app name.
+     You can also specify your app's language and SKU using `--language` and `--sku` params respectively. If you have never submitted any app before, you may also have to specify your company name using `--company-name` parameter.
+
+   > If you already have an App Store Connect app, this step can be skipped by providing the `--app-asc-id` param. The [ASC App Id](https://expo.fyi/asc-app-id) param can be either found on App Store Connect, or later during this command in the _Submission Summary_ table.
+
+3. Ask for your Apple ID (if not provided earlier) and for your Apple app-specific password. They can be also provided using `--apple-id` param and `EXPO_APPLE_APP_SPECIFIC_PASSWORD` environment variable, respectively.
+4. Ask for binary to submit. You can select one of the following:
+
+   - The latest finished iOS build for the project on EAS servers
+   - Specific build ID. The ID can be found on your EAS Account under _Builds v2_ section
+   - Path to an `.ipa` archive on your local filesystem
+   - URL to the app archive
+
+   > This step can be skipped, if one of the following parameters is provided: `--latest`, `--id`, `--path` or `--url`.
+
+5. The summary of provided configuration is displayed and the submission process begins. The submission progress is displayed on the screen.
+6. Your build should now be visible on [App Store Connect](https://appstoreconnect.apple.com). If something goes wrong, an appropriate message is displayed on the screen.
+
+## Submitting your app using CI
+
+The `eas submit` command is able to perform submissions from a CI environment. All you have to do is ensure that all required information is provided with command parameters and environment variables. Mainly, providing the archive source (`--latest`, `--id`, `--path` or `--url`) is essential. Also, make sure that Android package and/or iOS Bundle Identifier is present in your app configuration file, unless overwritten explicitly with flag parameters.
+
+#### iOS submissions
+
+For iOS submissions you have to provide `EXPO_APPLE_APP_SPECIFIC_PASSWORD` environment variable along with Apple ID (`--apple-id`) and ASC App Id (`--asc-app-id`) parameters. The last one is required to skip Apple developer login with potential prompt for 2FA authentication.
```

Comment:
>  The last one is required to skip Apple developer login with potential prompt for 2FA authentication.

is this referring to the `asc-app-id`? we should be explicit and name it in this sentence for clarity

Reply:
Yes, I've clarified it.